OrbitQt/outputdialog.cpp

```diff
@@ -23,7 +23,7 @@ void OutputDialog::SetStatus(const std::string& a_Status) {
 }
 
 //-----------------------------------------------------------------------------
-void OutputDialog::AddLog(const std::wstring& a_Log) {
-  std::wstring log = ui->OutputTextEdit->toPlainText().toStdWString() + a_Log;
-  ui->OutputTextEdit->setPlainText(QString::fromWCharArray(log.c_str()));
-}
\ No newline at end of file
+void OutputDialog::AddLog(const std::string& a_Log) {
+  std::string log = ui->OutputTextEdit->toPlainText().toStdString() + a_Log;
+  ui->OutputTextEdit->setPlainText(log.c_str());
```

Comment:
`QString::fromStdString` would be better here.

Reply:
Or maybe even convert a_Log to a QString and make the concatenation in QString-land. I'm not really sure which is nicer/faster/cleaner.

Reply:
setPlainText expects QString?

Reply:
done